File: LoadingAndSaving/LoadingAndSavingTxt/LoadingAndSavingTxtExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.LoadingAndSaving.LoadingAndSavingTxt
{
    [TestClass, TestFixture]
    public class LoadingAndSavingTxtExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void LoadTxt()
        {
            TestHelper.SetDataDir("LoadingAndSaving/LoadingAndSavingTxt/LoadTxt");

            LoadTxtExample.Program.Main();
        }

    }
}

File: Examples/LoadingAndSaving/LoadingAndSavingTxt/LoadingAndSavingTxtExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.LoadingAndSaving.LoadingAndSavingTxt
	<TestClass, TestFixture> _
	Public Class LoadingAndSavingTxtExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub LoadTxt()
	            TestHelper.SetDataDir("LoadingAndSaving/LoadingAndSavingTxt/LoadTxt")
	
	            LoadTxtExample.Program.Main()
        End Sub

	End Class
End Namespace